gcp/compute/helpers.py

```diff
@@ -1,3 +1,36 @@
+def does_firewall_open_all_ports_to_any(firewall):
+    """
+    Returns True if firewall has a rule to open all ports to any source. Excludes ICMP.
+
+    >>> does_firewall_open_all_ports_to_any({})
+    False
+    >>> does_firewall_open_all_ports_to_any({'sourceRanges': ['1.1.1.1/1'], 'allowed': [{'ports': ['1', '2', '300']}]})
+    False
+    >>> does_firewall_open_all_ports_to_any({'sourceRanges': ['1.1.1.1/1'], 'allowed': [{'ports': ['0-65535']}]})
+    True
+    >>> does_firewall_open_all_ports_to_any({'sourceRanges': ['0.0.0.0/0'], 'allowed': [{'ports': ['0-65535']}]})
+    True
+    >>> does_firewall_open_all_ports_to_any({'sourceRanges': ['10.0.0.5/32'], 'allowed': [{'ports': ['0-65535']}]})
+    True
```

Comment:
Do we need to worry about IPv6?

Reply:
Not for these tests, but thanks for bringing this up! Needed to confirm that. IPv6 is not supported within VPC networks, but is supported for load balancers, so if/when tests are added for those IPv6 will need to be worried about.